src/wp-admin/js/theme-plugin-editor.js

```diff
@@ -90,7 +90,6 @@ wp.themePluginEditor = (function( $ ) {
 	 * @return {void}
 	 */
 	component.showWarning = function() {
-		var oneSecond = 1000;
```

Comment:
`var oneSecond = 1000;`

Leave this line in the code to silence `No magic number: 1000` errors / warnings in ESLint, with line 114 left as:

`}, oneSecond );`

Reply:
Thanks for the explanation! Now done.